### PR TITLE
Add documents of new td* operators

### PR DESCRIPTION
**Pull request by frsyuki**
td_for_each, td_table_wait, td_wait, and td_partial_delete.

### CODE REVIEW

File: digdag-docs/src/operators.rst [hunk 1]
```diff
@@ -429,13 +426,198 @@ Output parameters
...
+
+  * :command:`td>: queries/step1.sql`
+
+:command:`database: NAME`
+  Name of a database.
+
+  * :command:`database: my_db`
+
+:command:`apikey: APIKEY`
+  API key. You can set this at command line using ``-p td.apikey=$TD_APIKEY`` argument.
```
Comment (danielnorberg) [inline digdag-docs/src/operators.rst]: Can we stop telling people to set apikey using params?

File: digdag-docs/src/operators.rst [hunk 2]
```diff
@@ -429,13 +426,198 @@ Output parameters
...
+
+  * :command:`rows: 10`
+
+:command:`database: NAME`
+  Name of a database.
+
+  * :command:`database: my_db`
+
+:command:`apikey: APIKEY`
+  API key. You can set this at command line using ``-p td.apikey=$TD_APIKEY`` argument.
```
Comment (danielnorberg) [inline digdag-docs/src/operators.rst]: Can we stop telling people to set apikey using params?

File: digdag-docs/src/operators.rst [hunk 3]
```diff
@@ -429,13 +426,198 @@ Output parameters
...
+td_wait>: Waits for data arriving at Treasure Data table
+----------------------------------
+
+**td_wait>:** operator runs a query periodically until it returns true. This operator can use more complex query compared to ``td_wait_table>:`` operator
+
+.. code-block:: yaml
+
+    _export:
+      td:
+        apikey: YOUR/API_KEY
```
Comment (danielnorberg) [inline digdag-docs/src/operators.rst]: Can we stop telling people to set apikey using params?

**Comment by danielnorberg** [comment]
Let's stop telling people to provide apikeys as plaintext params.

**Comment by frsyuki** [comment]
I wonder how we should document about apikey...secret parameters are not available in local mode.

**Comment by frsyuki** [comment]
@danielnorberg I rebased & updated about secrets.

**Pull request merged**